startup.py

```diff
@@ -214,8 +214,11 @@ def _get_installation_paths_from_registry(logger):
             sub_key_names.append(winreg.EnumKey(key, i))
             i += 1
         winreg.CloseKey(key)
-    except WindowsError:
-        logger.error("error opening key %s" % base_key_name)
+    except WindowsError as err:
+        logger.debug(
+            "error opening key %s : (%s) errno: %s, winerror: %s"
+            % (base_key_name, err.strerror, err.errno, err.winerror)
```

Comment:
You can simplify with:
```python
logger.debug(
    "error opening key %s : (%s) errno: %s, winerror: %s",
    base_key_name, err.strerror, err.errno, err.winerror,
)
```
as https://docs.python.org/3/library/logging.html#logging.Logger.debug perform % formating

If you want more information, you can even pass `exc_info=err` to the logger so it will print the TB.